src/std-indices/model.cmake

```diff
@@ -50,4 +66,13 @@ macro(setup)
         register_definitions(USE_ONEDPL)
         register_link_library(oneDPL)
     endif ()
+    if (CLANG_OFFLOAD)
```

Comment:
I'm not totally sold on the name of this definition as CLANG is not specific to AMD, but the `CLANG_FLAGS` set below are definitely specific. Can we rename this please to something appropriate?

Reply:
Would replacing `CLANG_OFFLOAD` with `AMDGPU_TARGET_OFFLOAD` and `CLANG_FLAGS` with `AMDGPU_TARGET_FLAGS` be acceptable?